### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by deepayan1**

### CODE REVIEW

**Comment by kingthorin** [comment]
Can we give you a hand with this?

**Comment by deepayan1** [comment]
Yes please. For some reason its not working. RegardsDeepayanOn 14-Feb-2023, at 11:23 PM, Rick M ***@***.***> wrote:﻿
Can we give you a hand with this?

File: README.md [hunk 1]
```diff
@@ -78,7 +78,7 @@ Jeroen Beckers, [OWASP Mobile Application Security](http://mas.owasp.org), Co-Au
 Shubham Palriwala, [OWASP Juice Shop](https://owasp-juice.shop) Core Team member<br>
 Pinaki Mondal, Open Source Security Tools Developer<br>
 Zsolt Imre, CTO at private company<br>
-
+Deepayan Chanda, Principal Cybersecurity Architect
```
Comment (kingthorin) [inline README.md:81]:
```suggestion
Deepayan Chanda, Principal Cybersecurity Architect<br>

```

**Review by kingthorin — APPROVED** [review]

**Comment by deepayan1** [comment]
Thanks for helping
Any idea why it was not getting merge?

**Comment by deepayan1** [comment]
ok cool. Thanks.

**Pull request merged**